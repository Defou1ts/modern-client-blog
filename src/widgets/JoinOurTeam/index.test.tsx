import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { JoinOurTeam } from './index';

describe('JoinOurTeam', () => {
	it('renders a  JoinOurTeam', () => {
		render(<JoinOurTeam />);
		const text = screen.getByText('about.join-our-team.title');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('JoinOurTeam snapshot', () => {
		const component = renderer.create(<JoinOurTeam />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
