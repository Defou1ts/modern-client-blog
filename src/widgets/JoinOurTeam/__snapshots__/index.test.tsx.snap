// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JoinOurTeam JoinOurTeam snapshot 1`] = `
<div
  className="wrapper"
>
  <h2
    className="h h2 title"
  >
    about.join-our-team.title
  </h2>
  <p
    className="p medium text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
  </p>
  <a
    className="link"
    href="/contact"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    <button
      className="button primary"
    >
      about.join-our-team.button
    </button>
  </a>
</div>
`;
